app/renderer/js/notification/index.js

```diff
@@ -20,8 +20,14 @@ if (process.platform === 'darwin') {
 
 window.addEventListener('load', () => {
 	// Call this function only when user is logged in
-	// eslint-disable-next-line no-undef, camelcase
-	if (page_params.realm_uri) {
+	let webpageParams = null;
+	try {
+		// eslint-disable-next-line no-undef, camelcase
+		webpageParams = page_params;
+	} catch (err) {
+		webpageParams = null;
+	}// eslint-disable-next-line no-undef, camelcase
```

Comment:
Can you move the comment to the next line?

Reply:
@abhigyank  Done :+1: